client/src/pages/CookiesPolicy.js

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+import styled from "styled-components";
+
+import { theme } from "../constants/theme";
+const { colors } = theme;
+
+const PolicyContainer = styled.div`
```

Comment:
I notice some of the styles are similar across pages. Perhaps we could create a file that specifically exports styles used across the three components, and then import them in components where needed. What do you think?

Reply:
I like this idea!

Reply:
Yea I have no strong preference since the styling for these pages are rather minimal. But it's definitely good practice to reuse code wherever possible